versioned_docs/version-0.9.1/Contributing/CTRLPanel Test.md

```diff
@@ -4,45 +4,48 @@ sidebar_position: 4
 
 :::info
 
-Please only use CTRLPanel installations that are not modifyed by plugins/modules/scripts or themes.
+Please only use Ctrlpanel installations that are not modifyed by plugins/modules/scripts or themes.
 
 :::
 
-This repository is using the playwright framework to provide E2E to help developers of the project to speed up development and provide a more stable and secure product.    
-In case you wanna help by adding some tests, playwright got a awsome documentaion [here](https://playwright.dev/docs/intro) but here are some basics we already took care of so you can start right away.  
+This repository is using the Playwright framework to provide E2E to help developers of the project speed up development and provide a more stable and secure product.    
+If you want to support and help us by adding tests, Playwright has a awesome documentation which you can find [here](https://playwright.dev/docs/intro), but below are some provided examples of basic Playwright testing incase you want to skip the documentation.  
   
 ## Setup  
 ### Requirements  
-- A setup version of CTRLPanel with the version you wanna test  
-- A admin account & a user account on this panel  
-- Basic knowlage of JS (or TS)
-- NodeJS 18+  
+- A version of Ctrlpanel that's already set up.
+- A administrator & client account on your dashboard
```

Comment:
change A to An

Reply:
is it only me or does that not sound right

Reply:
im talking about L17

Reply:
never mind i forgot to add `account` after `administrator`